core_backend/app/schemas.py

```diff
@@ -70,8 +71,9 @@ class ContentCreate(BaseModel):
     Pydantic model for content creation
     """
 
-    content_metadata: dict = {}
+    content_title: str
```

Comment:
Would be good to have a script that updates this for existing data. I get an error since I have content from before

Reply:
But if it's just for me then don't worry about it. I can recreate my data.

Reply:
I was going to deal with this l later, even though I knew it should be part of the PR!!!!! I will add this to the description. It would be nice to have migration for Qdrant, too, but resorting to a script for now.